ssh-scraper/utils.py

```diff
@@ -4,7 +4,9 @@
 from sqlalchemy.orm import Session
 from sqlalchemy import and_
 import copy
+from dataclasses import dataclass
 
+@dataclass
```

Comment:
So dataclasses actually create the constructor and __repr__ methods for you, and all you have to do is define the properties. I'll link the docs again
https://docs.python.org/3/library/dataclasses.html

but basically you new class for something like timeblock can just be:
```
@dataclass
class Timeblock:
   course_id: str
   section: str
   #other properties here
```
and it will create the constructor and repr for you